dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JSONEventsTest.java

```diff
@@ -211,11 +257,11 @@ private void checkJsonMessage(String line, ArrayList<String> mandatoryKeyList, A
         }
         if (mandatoryKeyList.size() > 0) {
             Log.info(c, method, "Mandatory keys missing: " + mandatoryKeyList.toString());
-            Assert.fail("Mandatory FFDC keys missing: " + mandatoryKeyList.toString() + ". Actual JSON was: " + line);
+            Assert.fail("Mandatory keys missing: " + mandatoryKeyList.toString() + ". Actual JSON was: " + line);
         }
         if (invalidFields.size() > 0) {
             Log.info(c, method, "Invalid keys found: " + invalidFields.toString());
-            Assert.fail("Invalid FFDC keys found: " + invalidFields.toString() + ". Actual JSON was: " + line);
```

Comment:
This and above were probably copy and pasted before. This method isn't specific to FFDC keys.